src/wallet/rpcwallet.cpp

```diff
@@ -1141,13 +1167,35 @@ UniValue addmultisigaddress(const JSONRPCRequest& request)
     if (!request.params[2].isNull())
         strAccount = AccountFromValue(request.params[2]);
 
+    int required = request.params[0].get_int();
+
+    // Get the public keys
+    const UniValue& keys = request.params[1].get_array();
```

Comment:
Suggestion: name this variable `keys_or_addresses`

Reply:
Renamed to `keys_or_addrs`